code/modules/admin/admin.dm

```diff
@@ -1175,9 +1175,8 @@ datum/admins/var/obj/item/weapon/paper/admin/faxreply // var to hold fax replies
 		P.stamps += "<hr><i>This paper has been stamped by the [P.origin] Quantum Relay.</i>"
 
 		var/image/stampoverlay = image('icons/obj/bureaucracy.dmi')
-		var/{x; y;}
```

Comment:
this is some wierd syntax, are you sure its not valid?

Reply:
Byond may allow compiling in this format but I try to make everything conform to the standard to which we write code.